app/controllers/FileChecksResultsController.scala

```diff
@@ -30,8 +30,9 @@ class FileChecksResultsController @Inject()(val controllerComponents: SecurityCo
         )
         Ok(views.html.standard.fileChecksResults(consignmentInfo, consignmentId, request.token.name))
       } else {
-        Ok(views.html.fileChecksFailed(request.token.name, isJudgmentUser = false))
-      }
+        val fileStatus = fileCheck.files.find(!_.fileStatus.contains("Success")).get.fileStatus.get
```

Comment:
Wondering what should happen if there are both zips and password protected files found? I think that is a question for Kirsten and Sarra. Not sure there are designs for such a case? I think the `fileStatus` should be a list to pass into the template.